spec/openapi.yml

```diff
@@ -503,7 +503,14 @@ paths:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/newTag'
+              $ref: '#/components/schemas/CreatedTag'
+      responses:
+        '201':
```

Comment:
Since this is a `PUT` call, we return a `200`, see [`/api/v1/tags/{name}`](https://github.com/MarquezProject/marquez/blob/main/api/src/main/java/marquez/api/TagResource.java#L69)

Reply:
Hmm, per the HTTP spec, we should return a 201 if a resources is created and a 200 if it already exists. I can add both options to the response here.

Reply:
I see, currently we only return `200`s for all `PUT` calls. So, I'd prefer we have the spec aligned with what's implemented. Mind opening an issue to update our `PUT` calls to return `200` for updates and `201` for new resources?